charts/akeyless-secure-remote-access/Chart.yaml

```diff
@@ -16,9 +16,11 @@ type: application
 
 version: 0.24.5
 
-appVersion: v0.20.3_0.15.7
+
+appVersion: v0.20.2_0.15.7_1.2.5
```

Comment:
I think that if you don't edit this release helm script here:
 .github/scripts/latest_version_release.sh
with ZTP annotations on SRA then it will break once someone will release ssh bastion or ssh proxy.
maybe you can edit it with a fixed version for now until you will finish the workflow release on frontend repo

Reply:
done